terraform/projects/prom-ec2/paas-production/extra-prometheus-scrape-configs.yml.tpl

```diff
@@ -1,3 +1,27 @@
+- job_name: dcs-federate
+  scheme: https
+  honor_labels: true
+  honor_timestamps: true
+  metrics_path: '/federate'
```

Comment:
Oh. How long were you going to wait for me to realize I'd mapped the wrong route path on the internal prometheus (I thought it used `/metrics` like anything else)